sshcode.go

```diff
@@ -608,3 +629,27 @@ func parseGCPSSHCmd(instance string) (ip, sshFlags string, err error) {
 
 	return strings.TrimSpace(userIP), sshFlags, nil
 }
+
+// When the user passes a path such as `/Workspace`, msys2 gives sshcode `C:/msys64/Worksapce`, which is not a valid remote path
```

Comment:
```suggestion
// gitbashWindowsDir strips a the msys2 install directory from the beginning of
// the path. On msys2, if a user provides `/workspace` sshcode will receive
// `C:/msys64/workspace` which won't work on the remote host.
```

Function comments should always be in the form `// functionName ...`. Comment should also be wrapped to 80 chars.

Reply:
alright, makes sense,